forge-gui/res/cardsfolder/upcoming/thran_power_suit.txt

```diff
@@ -1,9 +1,11 @@
 Name:Thran Power Suit
 ManaCost:2
 Types:Artifact Equipment
-S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddToughness$ X | AddPower$ X | AddKeyword$ Ward:2 | Description$ Equipped creature gets +1/+1 for each Aura and Equipment attached to it and has Ward {2}. (Whenever equipped creature becomes the target of a spell or ability an opponent controls, counter it unless that player pays {2}.)
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddToughness$ X | AddPower$ X | AddKeyword$ Ward:2 | AddSVar$ EquipMe & EnchantMe | Description$ Equipped creature gets +1/+1 for each Aura and Equipment attached to it and has Ward {2}. (Whenever equipped creature becomes the target of a spell or ability an opponent controls, counter it unless that player pays {2}.)
+SVar:EquipMe:SVar:EquipMe:Multiple
```

Comment:
dual svar on single line?

Reply:
It's to add `SVar:EquipMe:Multiple` to the Affected. Many cards like Chainflail Centipede have similar.